core/src/main/kotlin/org/evomaster/core/problem/external/service/ExternalServiceInfo.kt

```diff
@@ -1,10 +1,15 @@
 package org.evomaster.core.problem.external.service
 
+import com.github.tomakehurst.wiremock.WireMockServer
+
 class ExternalServiceInfo(
         val protocol: String,
         val remoteHostname: String,
         val remotePort: Int) {
 
+    // TODO: Need to refactor
+    private lateinit var wireMockServer : WireMockServer
```

Comment:
not sure should be done here in a class called `Info`. Maybe create a new class to deal with the actual storage and handling of WM instances

Reply:
or, if keeping here, likely need new name for this class

Reply:
Original I had this class name as `ExternalService`, later we changed it to `Info` because it was confusing with `ExternalServices`.